docs/source/getting-started.md

```diff
@@ -0,0 +1,21 @@
+# Getting started
+
+`menuinst` can be used to create shortcuts or menu items across operating systems.
+It's designed to integrate nicely with `conda` packages, 
```

Comment:
How does menuinst relate to mamba. Will it also be used in that case 🤔

Reply:
I think "yes" on `mamba`, because it delegates transactions to `conda`, not on `micromamba` (yet). This will change soon though.

Reply:
Added a note on what issues to follow.